shop.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shop Philips</title>
+</head>
+<body>
+    <header>
+        <!-- ссылка и окно поиска -->
+        <a href="/">Shop </a>
+        <input type="search"  id="name" name="Your name"  value=""> 
+        <input type="submit" value="Search"> 
+        <hr>
+        <nav>
+            <a href="#photo&video">Photo & Video</a> |
+            <a href="#changeTheWayYouUseLight">Change The Way You Use Light</a> |
+            <a href="#productInformation">Product Information</a>
+        </nav>
+        <hr>
+    </header>
+    <main>
+        <section>
+          <h1>
+            Philips Hue White and Color Ambiance A19 60W Equivalent LED Smart Bulb Starter Kit
+          </h1> 
+          Price: &#8364; 100  <input type="button" value="Buy now">
+          <ul>
+              <li>
+                 <a href="#">Order via email buy@shop.com</a> 
+              </li>
+              <li>
+                  <a href="#">Order via call (111)1111111</a>
+              </li>
+          </ul> 
+        </section>
+
+        <section>
+            <h1>
+                Photo and Video
+            </h1>
```

Comment:
```suggestion
            <h2>
                Photo and Video
            </h2>
```

`<h1>` может быть только один на странице